ui/brave_ads/ad_notification_view_md.cc

```diff
@@ -403,9 +403,6 @@ void AdNotificationViewMD::UpdateControlButtonsVisibilityWithNotification(
   UpdateControlButtonsVisibility();
 }
 
-void AdNotificationViewMD::ButtonPressed(views::Button* sender,
```

Comment:
Were changes in `ui/brave_ads` intentional as these will likely break ads?

Reply:
Yes, these are due to the Chromium change: https://chromium.googlesource.com/chromium/src/+/37cb433bb3d86231ee0c37cdd8a08f31ab35f172 (views::ButtonListener has been removed).